torch/_inductor/lowering.py

```diff
@@ -1172,9 +1172,110 @@ def permute(x, dims):
 
 @register_lowering(aten.slice, type_promotion_kind=None)
 def slice_(x, dim=0, start=0, end=2**63, step=1, clamp=True):
+    """
+    Lowers a slice call, creating ExternKernels for the output size & storage offset symbols,
+    if the indices are unbacked and appropriate semantics aren't known.
```

Comment:
nit: semantics are the same for known and unknown. 

maybe:
creating ExternKernels to compute the output size & storage offset symbols dynamically if they  cant be determined at compile time because indices are unbacked.